haicrypt/cryspr-openssl-evp.c

```diff
@@ -247,6 +251,83 @@ int crysprOpenSSL_EVP_AES_CtrCipher(bool                 bEncrypt, /* true:encry
     return 0;
 }
 
+int crysprOpenSSL_EVP_AES_GCMCipher(bool                 bEncrypt, /* true:encrypt, false:decrypt */
```

Comment:
Hm, I did it a different way: by introducing the `HaiCryptCryspr_Is_AES_GCM_Supported(void)` function that is visible to SRT as well, so it does not accept GCM-configured connections if the crypto library does not support it.
In fact, with `CRYSPR_HAS_AESGCM` it might be easier and more consistent.